### DISCUSSION TITLE
Extract math equations

### DISCUSSION THREAD

**Discussion by erkin98** [jsvine/pdfplumber#402] [Ask for help with specific PDFs]
is it possible extract equations from [pdf ](https://drive.google.com/file/d/16UJFnaQnhtQNscwyLL6S1gfF4Bly4EjF/view?usp=sharing) exactly ?
page 82-84 in book

**Comment by samkit-jain** [comment]
I think the reason you are not able to extract the mathematical text correctly is that the PDF is a scanned document with a text layer created by an OCR software. You'll notice that a lot of the text is not captured (copyable) and there are quite a few inconsistencies as well like B being recognised as R or curves being recognised as J, ...
